backend/alert/management/commands/statusaudit.py

```diff
@@ -0,0 +1,98 @@
+import json
+import logging
+
+from django.core.management.base import BaseCommand
+from tqdm import tqdm
+
+from alert.models import Course, Section
+from courses import registrar
+from courses.util import get_course_and_section, get_current_semester, translate_semester_inv
+
+
+class Command(BaseCommand):
+    help = """
+        Generate an audit report that demonstrates the differences between our
+        database and the course statuses received from using the OpenData
+        endpoint directly.
+
+        Note that this script DOES NOT make any changes to the database, just
+        generates a textfile report
+    """
+
+    def handle(self, *args, **options):
+        root_logger = logging.getLogger("")
+        root_logger.setLevel(logging.DEBUG)
+
+        semester = get_current_semester()
+        statuses = registrar.get_all_course_status(semester)
+        stats = {
+            "missing_data": 0,
+            "section_not_found": 0,
+            "duplicate_updates": 0,
+            "unsynced_updates": 0,
+        }
+        unsynced_courses = []
+        for status in tqdm(statuses):
+            data = status
+            section_code = data.get("section_id_normalized")
+            if section_code is None:
+                stats["missing_data"] += 1
+                continue
+
+            course_status = data.get("status")
```

Comment:
is this status per section or per course?

Reply:
It should be `section_status`, will change.